ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -206,7 +206,7 @@ private void UpdateFlags()
                 ModelState.Remove("ClinicalDetails.SymptomStartDate");
             }
 
-            if (ClinicalDetails.DidNotStartTreatment == true)
+            if (ClinicalDetails.StartedTreatment != true)
```

Comment:
Careful! If the value is `null` then the behaviour here has changed. We should be using `== false` instead.

Reply:
Yes, thanks for spotting it!